caribou/common/utils.py

```diff
@@ -64,3 +66,21 @@ def process_node(node: ast.AST, context: dict) -> None:
         process_node(node, context)
 
     return source_code
+
+
+def compress_json_str(json_str: str, compression_level: int = 21) -> bytes:
+    # Compress the JSON string using zstandard
+    json_bytes = json_str.encode("utf-8")
+    cctx = zstd.ZstdCompressor(level=compression_level)
```

Comment:
Do we document somewhere why this compression was chosen?

Reply:
This was not documented, but I will document this in a later PR. Refer to issue #340.